thoth/user_api/api_v1.py

```diff
@@ -414,6 +414,36 @@ def list_python_package_indexes():
     return graph.python_package_index_listing()
 
 
+def list_hardware_environments(
+    is_external: bool = False,
```

Comment:
Remove this parameter - it's misleading as it is not propagated from openapi (the same for sw environment listing).

Reply:
Thanks, deleted from `openapi.yaml` and forgot to delete from `api_v1.py`.